include/z64bgcheck.h

```diff
@@ -15,24 +17,53 @@ struct DynaPolyActor;
 
 #define BGACTOR_NEG_ONE -1
 #define BG_ACTOR_MAX 50
+#define DYNA_WATERBOX_MAX 30
 #define BGCHECK_SCENE BG_ACTOR_MAX
 #define BGCHECK_Y_MIN -32000.0f
 #define BGCHECK_XYZ_ABSMAX 32760.0f
```

Comment:
What are your thoughts on adding a:
`#define BGCHECK_Y_MAX 32000.0f`
I recall being able to use that in one or two places, but it would be much less common than `BGCHECK_Y_MIN`

Reply:
implemented